charts/datree-admission-webhook/templates/namespace-post-delete.yaml

```diff
@@ -33,7 +33,7 @@ spec:
             - "-c"
             - >-
               kubectl delete validatingwebhookconfigurations.admissionregistration.k8s.io datree-webhook -n {{template "datree.namespace" .}};
-              kubectl label ns kube-system {{template "datree.namespace" .}} datree.io/skip-;
+              kubectl label ns kube-system {{template "datree.namespace" .}} admission.datree/validate-;
```

Comment:
Shouldn't we remove the label from kube-public and kube-node-lease as well?

Reply:
we absolutely should:) missed it in my commit